index.ts

```diff
@@ -10,14 +10,23 @@ app.get('/', (req: Request, res: Response) => {
 });
 
 // Get user
-app.get('/user/:id', (req: Request, res: Response) => {
+app.get('/user/:id', async (req: Request, res: Response) => {
     const id = req.params.id
 
-  const user = UserService.getUser(id);
+  const user = await UserService.getUser(id);
 
   res.send({ user });
 });
 
+// Get user first and last name
+app.get('/user/:id/firstLast', async (req: Request, res: Response) => {
+    const id = req.params.id
+
+  const u = UserService.getUserFirstAndLastName(id);
+
+  res.send({ u });
```

Comment:
```suggestion
  const userDetail = await UserService.getUserFirstAndLastName(id);

  res.send({ userDetails });
```
The naming for the variable is not very clear on what it is storing. I would suggest renaming it to userDetails instead
The function in the lib is an async function. I believe an await is missing here